pkg/pillar/hypervisor/kvm.go

```diff
@@ -459,6 +454,39 @@ func (ctx kvmContext) Task(status *types.DomainStatus) types.Task {
 	}
 }
 
+// TBD: Have a better way to calculate this number.
+// For now it is based on some trial-and-error experiments.
+func minVMMOverhead(config types.DomainConfig) int64 {
+	return 600 << 20
```

Comment:
If you really want to use bit shift here it's worth to let a comment (600 * 1024 * 1024). But since it's a constant expression (that's gonna be resolved during build time), why not let (600 * (1024 * 1024)) for better readability?

Reply:
I thought that power of two is much more clear notation than chain of multiplications (I always count "1024 once - makes kb, 1024 twice - makes mb, really annoying)  :) Will add a comment.